src/components/Review/ReviewList.js

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useState } from 'react';
+import '../../pages/Review/Review.scss';
+import ReviewListItem from './ReviewListItem';
+
+const ReviewList = () => {
+  const [reviewItem, setReviewItem] = useState([
+    {
+      id: '',
+      reviewTitle: '',
+      date: '',
+      userId: 'lemon',
+      userInput: '',
+    },
+  ]);
```

Comment:
처음부터 리뷰가 하나 들어가 있는게 실제 상황과 비교했을 때 적절하진 않은 것 같네요! 처음에는 리뷰가 비어있다가 나중에 데이터 통신을 통해서 실제 리뷰가 state에 채워지는 동작이 알맞아보입니다! 따라서, 초기값은 빈배열로 주는게 좋겠네요!

```suggestion
  const [reviewItem, setReviewItem] = useState([]);
```

Reply:
변수명에 Item등의 단어는 굳이 안들어가도 될 것 같네요 reviews 정도로만 해도 적절해보입니다!

Reply:
다른 부분도 유사한 사항 확인해주세요!